bin/load_scxa_analytics_index.sh

```diff
@@ -27,4 +27,11 @@ do
 done
 set -e
 rm $CHUNK_FILES
+
+curl -X POST -H 'Content-Type: application/json' \
```

Comment:
There is a potential here though for this to fail silently, if the commit fails and we don't catch the HTTP response code to asses if it failed or not. I think that we should catch that and fail with an exit code if the HTTP response falls in certain values.

Reply:
See code suggestions below.